### PR TITLE
Automation project use classic Page Object pattern

### PR DESCRIPTION
**Pull request by VictoriaBadenko**

### CODE REVIEW

File: src/main/java/yandex/mail/driverManager/DriverManager.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package yandex.mail.driverManager;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class DriverManager {
+    private static ThreadLocal<WebDriver> driver = new ThreadLocal<>();
```
Comment (LikBez123) [inline src/main/java/yandex/mail/driverManager/DriverManager.java]: make it final

File: src/main/java/yandex/mail/driverManager/DriverManager.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    public DriverManager() {
+    }
+
+    public static WebDriver getDriverInstance() {
+        if (driver.get() == null) {
+            WebDriverManager.chromedriver().setup();
```
Comment (LikBez123) [inline src/main/java/yandex/mail/driverManager/DriverManager.java]: you don't need it. Selenium already has an embedded WebDriverManager

File: pom.xml [hunk 3]
```diff
@@ -42,6 +42,11 @@
             <artifactId>selenium-chrome-driver</artifactId>
             <version>4.12.1</version>
         </dependency>
+        <dependency>
+            <groupId>io.github.bonigarcia</groupId>
+            <artifactId>webdrivermanager</artifactId>
+            <version>5.5.3</version>
+        </dependency>
```
Comment (LikBez123) [inline pom.xml]: i don't think you need it. Selenium already has an embedded WebDriverManager

**Review by LikBez123 — APPROVED** [review]
Please check and remove unused imports and fix the comments

**Review by victortylikov — APPROVED** [review]

**Pull request merged**